cli/azd/test/samples/funcapp/requirements.txt

```diff
@@ -1,3 +1,3 @@
-# Do not include azure-functions-worker as it may conflict with the Azure Functions platform
-
+# Do not include azure-functions-worker as it may conflict with the Azure Functions platform
```

Comment:
This change appeared and won't go away...

Reply:
This is a whitespace only change - I suspect it's line ending stuff.